docker-compose-image-tag.yml

```diff
@@ -22,9 +22,6 @@
 # unique random secure passwords and SECRET_KEY.
 # -----------------------------------------------------------------------
 x-superset-image: &superset-image apachesuperset.docker.scarf.sh/apache/superset:${TAG:-latest-dev}
-x-superset-depends-on: &superset-depends-on
```

Comment:
nit: We could keep the anchor and just update the values of it with the new conditions you have and also create a new one for the `superset-init: condition: service_completed_successfully` so we can then reuse and refer to these

Reply:
As much as I like DRY code, I felt like in yaml it was mostly just an indirection as there's not a lot that's repeated here. Personally prefer have it spelled out inline as opposed to having to look up a thing that has a few line and is repeated a few times.

Reply:
Could see it both ways though, let me know what you think

Reply:
Yeah, I don't have a strong preference, so we can kept it spelled out inline